src/ProductConstructionService/ProductConstructionService.BarViz/Code/Helpers/DateFormatter.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace ProductConstructionService.BarViz.Code.Helpers;
+
+public static class DateFormatter
+{
+    public static string FormatDate(DateTimeOffset date)
+    {
+        var now = DateTimeOffset.Now;
+        var timeFormat = "HH:mm";
+        var dateFormat = "yyyy-MM-dd";
+
+        if (date.Date == now.Date)
+        {
+            return $"Today at {date.ToString(timeFormat)}";
```

Comment:
Why not just put `date.ToString(timeFormat)` in a variable?

Reply:
done